api/pkg/transformer/types/table/table.go

```diff
@@ -367,7 +367,7 @@ func broadcastScalar(colMap map[string]interface{}, length int) map[string]inter
 				continue
 			}
 
-			val = colValueVal.Index(0)
+			val = colValueVal.Index(0).Interface()
```

Comment:
why change this?

Reply:
Because, when creating series of list elements, colValueVal.Index(0) returns reflect.Value instead of interface{}. So when we pass these reflect.Value to NewInferType to create the series, it will use wrong series' type [1]->[2], because it sends reflect.Struct instead of the correct slice type.

[1] https://github.com/gojek/merlin/blob/support-list-features/api/pkg/transformer/types/series/series.go#L60-L66
[2] https://github.com/gojek/merlin/blob/support-list-features/api/pkg/transformer/types/series/series.go#L238-L260